.vscode/settings.json

```diff
@@ -41,6 +41,8 @@
         "xlocmon": "cpp",
         "xmemory": "cpp",
         "xstring": "cpp",
-        "xutility": "cpp"
+        "xutility": "cpp",
+        "iosfwd": "cpp",
+        "sstream": "cpp"
```

Comment:
Wy do these get added I wonder?

Reply:
not sure as well, but committing since it does not hurt it seems